concent_api/core/tests/utils.py

```diff
@@ -625,3 +647,24 @@ def _add_signature_to_message(self, golem_message, priv_key):  # pylint: disable
         golem_message = golem_message.serialize(sign_func = signature)
         golem_message = message.Message.deserialize(golem_message, decrypt_func = None, check_time = False)
         return golem_message.sig
+
+    def _create_client_auth_message(self, client_priv_key, client_public_key):  # pylint: disable=no-self-use
```

Comment:
this method is defined in to more places, please extract to common pleace and reuse it ;)